Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -682,6 +713,47 @@ namespace O3DE::ProjectManager
         
     }
 
+    void ProjectsScreen::HandleOpenProjectExportSettings(const QString& projectPath)
+    {
+        AZ::Outcome<EngineInfo> engineInfoResult = PythonBindingsInterface::Get()->GetProjectEngine(projectPath);
+        AZ::Outcome projectBuildPathResult = ProjectUtils::GetProjectBuildPath(projectPath);
+
+        auto engineInfo = engineInfoResult.TakeValue();
+        auto buildPath = projectBuildPathResult.TakeValue();
+
+        QString projectName = tr("Project");
+        auto getProjectResult = PythonBindingsInterface::Get()->GetProject(projectPath);
+        if (getProjectResult)
+        {
+            projectName = getProjectResult.GetValue().m_displayName;
+        }
+
+        const QString pythonPath = ProjectUtils::GetPythonExecutablePath(engineInfo.m_path);
+        const QString o3dePath = QString("%1/scripts/o3de.py").arg(engineInfo.m_path);
+
+
+        // Let's start the python script.
+        QProcess process;
+        process.setProgram(pythonPath);
+        const QStringList commandArgs{ o3dePath, "export-project", "-pp", projectPath, "--configure" };
+
+        process.setArguments(commandArgs);
+
+        // It's important to dump the command details in the application log so the user
+        // would know how to spawn the Android Project Generator from the command terminal
+        // in case of errors and debugging is required.
+        const QString commandArgsStr = QString("%1 %2").arg(pythonPath, commandArgs.join(" "));
+        AZ_Printf(
+            "ProjectManager",
+            "Will start the Android Project Generator with the following command:\n%s\n",
```

Comment:
Android??

Reply:
this was copied from your prototype branch, so this was an overlooked copy/paste issue that i missed. fixing it now